server/src/routes/api/v1/playersRouter.js

```diff
@@ -14,10 +14,7 @@ playersRouter.get("/", (req, res) => {
     } else {
       const parsedResponse = JSON.parse(data)
       const player = PlayerSerializer.getSummary(parsedResponse.data[0])
-      res
-        .set({ "Content-Type": "application/json" })
-        .status(200)
-        .json(player)
+      res.set({ "Content-Type": "application/json" }).status(200).json(player)
```

Comment:
curious why you need to specify the content-type? saying `.json(player)` should cover that for us